test/cli/test_detect.py

```diff
@@ -190,6 +189,47 @@ def test_oslogin_proto_beta(self):
                 "google/cloud/oslogin/v1beta/oslogin.proto L179: Field behavior of an existing field `ssh_public_key` is changed.\n",
             )
 
+    def test_pubsub_proto(self):
+        with patch("sys.stdout", new=StringIO()):
+            runner = CliRunner()
+            result = runner.invoke(
+                detect,
+                [
+                    f"--original_api_definition_dirs=googleapis/,{self.COMMON_PROTOS_DIR}",
+                    f"--update_api_definition_dirs=googleapis/,{self.COMMON_PROTOS_DIR}",
+                    "--original_proto_files=googleapis/google/pubsub/v1beta2/pubsub.proto",
+                    "--update_proto_files=googleapis/google/pubsub/v1/pubsub.proto",
+                    "--human_readable_message",
+                ],
+            )
+            self.assertEqual(result.exit_code, 0)
+            self.assertEqual(
+                result.output,
+                "google/pubsub/v1/pubsub.proto L160: Field behavior of an existing field `name` is changed.\n"
+                + "google/pubsub/v1/pubsub.proto L221: Field behavior of an existing field `topic` is changed.\n"
+                + "google/pubsub/v1/pubsub.proto L245: Field behavior of an existing field `topic` is changed.\n"
+                + "google/pubsub/v1/pubsub.proto L250: Field behavior of an existing field `messages` is changed.\n"
+                + "google/pubsub/v1/pubsub.proto L266: Field behavior of an existing field `project` is changed.\n"
+                + "google/pubsub/v1/pubsub.proto L267: A resource reference option is added to the field `project`, but it is not defined anywhere\n"
```

Comment:
The resource ref `"cloudresourcemanager.googleapis.com/Project"` is not defined in api-common-protos or API definition files, it is in `google3/google/cloudresourcemanager/v1/projects.proto` which is not in third_party folder so we do not have a way to include it as a protoc folder. It is a known resource lost breaking change. Same as below.

Reply:
This resource is defined here:
https://github.com/googleapis/googleapis/blob/master/google/cloud/common_resources.proto
So this proto file should be included when running the detector for cloud APIs.

Reply:
I see, adding the `common_resources.proto` helps remove the errors! Thank you!